.gitignore

```diff
@@ -3,6 +3,6 @@
 .phpstorm.meta.php
 composer.lock
 /vagrant/
-/config.d/
+/.driver/
```

Comment:
This shouldn't be gitignored.
```suggestion
```

Reply:
@prince-swiftotter This is not custom config. This is library config. The gitignore file is for the library, not for the root project :-).

Reply:
Yes, I mean library config. Well my doubt is clear, I was thinking in a wrong way. Thanks!